test/distributed/tensor/parallel/test_fsdp_2d_parallel.py

```diff
@@ -145,6 +145,15 @@ def _apply_optim_in_backward(param_group):
     )
 
 
+def _clean_up_fsdp_param_name(name):
```

Comment:
Curious: Is this different from `clean_tensor_name()`?
https://github.com/pytorch/pytorch/blob/11c6a98bcae9e5ae305f8b21b9180a5ee2e1ba71/torch/distributed/fsdp/_common_utils.py#L228

Reply:
Thanks! I was not aware of this in the _common_utils. Will re-use then!